tests/compile-fail/fs/unix_open_missing_required_mode.rs

```diff
@@ -0,0 +1,20 @@
+// ignore-windows: No libc on Windows
+// compile-flags: -Zmiri-disable-isolation
+
+#![feature(rustc_private)]
+
+extern crate libc;
+
+fn main() {
+    #[cfg(unix)]
```

Comment:
We already have `ignore-windows`, why is there also a `cfg(unix)`?

Reply:
This should probably just be `only-unix` if that's a thing.  This change only affects the unix code path, but I don't exactly know how to set the tests up to only test unix.

Reply:
"ignore-windows" is how we express "only-unix" in the Miri test suite. ;) It's not pretty but it works... for now.